src/steps/generic.rs

```diff
@@ -479,31 +479,33 @@ pub fn run_composer_update(ctx: &ExecutionContext) -> Result<()> {
 pub fn run_dotnet_upgrade(ctx: &ExecutionContext) -> Result<()> {
     let dotnet = utils::require("dotnet")?;
 
-    let output = Command::new(dotnet)
-        .args(["tool", "list", "--global"])
-        .output_checked_utf8()?;
-
-    if !output.stdout.starts_with("Package Id") {
-        return Err(SkipStep(String::from("dotnet did not output packages")).into());
-    }
+    let dotnet_tool_output = ctx.run_type().execute(&dotnet).arg("tool").output().err().unwrap();
+    if dotnet_tool_output.to_string().contains("Usage") {
```

Comment:
`dotnet` will display the localized text if I didn't remember wrong. When the localized text is shown, this condition may be failed.

Reply:
I think I know how to solve that.

Reply:
Should work now because it now queries one line which is identical in every language